src/components/InteractionLayer/index.js

```diff
@@ -545,23 +545,29 @@ class InteractionLayer extends React.Component {
 
       let s = null;
 
-      if (a.start.xval) {
-        scaledArea.start.xpos = xScale(a.start.xval);
-      }
-      if (a.end.xval) {
+      scaledArea.start.xpos = xScale(a.start.xval);
+
+      if (a.end !== undefined) {
         scaledArea.end.xpos = xScale(a.end.xval);
       }
 
       if (a.seriesId) {
         s = series.find(s1 => s1.id === a.seriesId);
         if (s) {
-          const { [Axes.y]: ySubDomain } = subDomainsByItemId[s.id];
+          const { [Axes.y]: ySubDomain } = subDomainsByItemId[
+            s.collectionId || s.id
+          ];
           const yScale = createYScale(ySubDomain, height);
-          if (a.start.yval) {
-            scaledArea.start.ypos = yScale(a.start.yval);
-          }
-          if (a.end.yval) {
+
+          scaledArea.start.ypos = yScale(a.start.yval);
+
+          if (a.end !== undefined) {
             scaledArea.end.ypos = yScale(a.end.yval);
+          } else {
+            scaledArea.end = {
```

Comment:
So if end isn't defined, it will set it to the bottom right corner? Interesting.
Right now this doesn't seem very consistent. Perhaps instead we should do checks to see if each individual value is defined, and if not, set them to a default?

e.g. instead of checking for .end to be defined, check each pos inside of .end and set them accordingly. and do the same for .start

Reply:
It sets it to the top right corner. Checking for each position individually could be more flexible. Are you okay with the defaults of start being bottom left and end being top right?

Reply:
I went ahead and did this ^ 😅. If start is `undefined` you get the bottom left point, if end is `undefined` you get top right and if they are defined but they are missing a `xpos` or `ypos` it gets filled in with the default start or end `xpos` or `ypos`.

Reply:
🤔 
I would expect something like:
start.ypos === yScale(start.ypos || ySubDomain[0])
start.xpos === yScale(start.ypos || timeDomain[0])
end.ypos === yScale(start.ypos || ySubDomain[1])
end.xpos === yScale(start.ypos || timeDomain[1])

e.g. top left = start, bottom right = end. This seems natural to me atleast

Reply:
This way as well we won't have infinitely sized areas, which will remove the need for setting values to Number.MIN/MAX_SAFE

Reply:
The changes I've already made use `scaledArea.start.ypos = yScale(area.start.ypos || ySubDomain[0])` and so on. It also checks and sets the whole start coordinate (`xpos` and `ypos`) if `area.start === undefined` and same for end.

My reasoning for using start as bottom-left and end as top-right is that the domain is defined that way already. The first value in an array we pass as a `Domain` is smaller and to the left or bottom. So I think it's more natural to set end to top-right. I can swap it if you think it's very unnatural tho 😅

Reply:
I find it strange but maybe I'm being weird. We should check with some more end users of griff (aka devs) to see what they think. Its a defining part of the API which we don't want to change in the future.

Check in with @evancharlton  @valgussev and @Meemaw to see what they think

Reply:
@evancharlton suggested we move away from start and end and instead defined `yMin`, `yMax`, `xMin` and `xMax`. That way there's no confusion

Reply:
I went ahead and implemented @evancharlton's suggestion. This is a breaking change so when we bump Griff in Insight we will have to update how we use areas.
Areas now look like this:
```
{
  id: ItemId,
  seriesId: ItemId,
  color?: string,
  xMin?: number,
  xMax?: number,
  yMin?: number,
  yMax?: number,
}
```